java/runtime/src/main/java/org/ray/runtime/generated/TaskInfo.java

```diff
@@ -38,23 +38,20 @@
   public ByteBuffer actorHandleIdInByteBuffer(ByteBuffer _bb) { return __vector_in_bytebuffer(_bb, 20, 1); }
   public int actorCounter() { int o = __offset(22); return o != 0 ? bb.getInt(o + bb_pos) : 0; }
   public boolean isActorCheckpointMethod() { int o = __offset(24); return o != 0 ? 0!=bb.get(o + bb_pos) : false; }
-  public String functionId() { int o = __offset(26); return o != 0 ? __string(o + bb_pos) : null; }
-  public ByteBuffer functionIdAsByteBuffer() { return __vector_as_bytebuffer(26, 1); }
-  public ByteBuffer functionIdInByteBuffer(ByteBuffer _bb) { return __vector_in_bytebuffer(_bb, 26, 1); }
   public Arg args(int j) { return args(new Arg(), j); }
-  public Arg args(Arg obj, int j) { int o = __offset(28); return o != 0 ? obj.__assign(__indirect(__vector(o) + j * 4), bb) : null; }
-  public int argsLength() { int o = __offset(28); return o != 0 ? __vector_len(o) : 0; }
-  public String returns(int j) { int o = __offset(30); return o != 0 ? __string(__vector(o) + j * 4) : null; }
-  public int returnsLength() { int o = __offset(30); return o != 0 ? __vector_len(o) : 0; }
+  public Arg args(Arg obj, int j) { int o = __offset(26); return o != 0 ? obj.__assign(__indirect(__vector(o) + j * 4), bb) : null; }
```

Comment:
This file seems to be automatically generated by flatbuffer. Why we need to keep it instead of generating it at installing? If the reason is that we need some patch, can we just create a patch file?

Reply:
@suquark  There is a function `returnsAsByteBuffer` which return the byte array. The code generated by flatbuffer will try to decode this byte array as a string, which will crash. Therefore, we currently modify this function by handle. This offset is not fixed and it needs to get changed according to the position of `returns`. A patch is not enough, a script can do this.

Reply:
I think we can file a PR to Flatbuffers to make the generated code expose the API we want.